gossip/src/cluster_info.rs

```diff
@@ -3224,92 +3206,6 @@ mod tests {
         ));
     }
 
-    #[test]
-    fn test_cluster_info_trace() {
```

Comment:
You didn't feel like keeping this one up to date anymore 😅 ?

Reply:
:smile: